tilelog/__init__.py

```diff
@@ -161,7 +167,7 @@ def app_logs(curs, date, dest):
     WHEN useragent LIKE 'flutter_map (%%)' THEN 'flutter_map (*)'
 
     -- Extract app name from foo.bar/123.456
-    WHEN regexp_like(useragent, '^([^./]+(\.[^./]+)*)/\d+(\.\d+)*$') THEN regexp_extract(useragent, '^([^./]+(\.[^./]+)*)/\d+(\.\d+)*$', 1) || '/*'
+    'WHEN regexp_like(useragent, '^([^./]+(\.[^./]+)*)/\d+(\.\d+)*$') THEN regexp_extract(useragent, '^([^./]+(\.[^./]+)*)/\d+(\.\d+)*$', 1) || '/*'
```

Comment:
It looks like a stray `'` was introduced

Reply:
I apologize for that, that was a complete mistake that I did not catch. It has been removed in my latest PR.